pkg/git/git_test.go

```diff
@@ -108,6 +113,198 @@ func TestChangeBranch(t *testing.T) {
 	})
 }
 
+func TestLogRange(t *testing.T) {
+
+	against := func(t *testing.T, r *git.Repository, from, to string, expected []plumbing.Hash) {
+		seen := []plumbing.Hash{}
+		cIter, err := LogRange(r, from, to)
+		if assert.NoError(t, err) {
+			err = cIter.ForEach(func(c *object.Commit) error {
+				seen = append(seen, c.ID())
+				return nil
+			})
+			if assert.NoError(t, err) {
+				if assert.Len(t, seen, len(expected)) {
+					assert.Subset(t, seen, expected)
+				}
+			}
+		}
+	}
+
+	prepareRepo := func() (r *git.Repository, hashes map[string]plumbing.Hash, err error) {
```

Comment:
https://github.com/src-d/go-git-fixtures keeps git repositories for testing purposes. a zipped repo in the test/resources could be an option.

Reply:
isn't the world beautiful? So many different ways and options for achieving a goal ...